### PR TITLE
Fix celery herd, simstart stuck phase, beat to 15s and no remove of p…

### PR DESCRIPTION
**Pull request by evetion**
…ending tasks.

### CODE REVIEW

File: delft3dworker/models.py [hunk 1]
```diff
@@ -1140,14 +1163,6 @@ def update_task_result(self):
             self.task_uuid = None
             self.save()
 
-        # Pending means not queued yet or unknown
```
Comment (evetion) [inline delft3dworker/models.py:1143]: REMOVED: If a tasks always runs longer than the heartbeat, we never get the result.

File: delft3dcontainermanager/tasks.py [hunk 2]
```diff
@@ -13,11 +14,13 @@
 logger = get_task_logger(__name__)
 
 
-@shared_task(bind=True)
+@shared_task(bind=True, base=QueueOnce, once={'graceful': True, 'timeout': 60})
```
Comment (evetion) [inline delft3dcontainermanager/tasks.py:17]: Timeout could be set to default celery task timeout.

File: delft3dworker/models.py [hunk 3]
```diff
@@ -498,42 +498,53 @@ def update_and_phase_shift(self):
...
-                desired_state='non-existent',
-            )
-            postprocess_container.save()
-            sync_clean_container = Container.objects.create(
-                scene=self,
-                container_type='sync_cleanup',
-                desired_state='non-existent',
-            )
-            sync_clean_container.save()
+            if not self.container_set.filter(container_type='preprocess').exists():
```
Comment (evetion) [inline delft3dworker/models.py:501]: Assures only one container is created, even if a Scene is put in phases.new again.

File: delft3dcontainermanager/tasks.py [hunk 4]
```diff
@@ -49,15 +52,17 @@ def get_docker_ps(self):
 
     client = Client(base_url='http://localhost:4000')
     containers = client.containers(all=True)  # filter here does not work
-    filtered_containers = [c for c in containers if c['Status'] not in ignore_states]
+    filtered_containers = [c for c in containers if c[
+        'Status'] not in ignore_states]
```
Comment (heijer) [inline delft3dcontainermanager/tasks.py:56]: does this newline cause no problems?

**Pull request merged**